frontend/src/components/activePage/TableInstance/TableInstance.tsx

```diff
@@ -54,6 +57,38 @@ const TableInstance: FC<ITableInstanceProps> = ({ ...props }) => {
 
   return (
     <>
+      <ModalAlert
+        headTitle="Destroy All"
+        alertMessage={<Text>Destroy All</Text>}
+        alertDescription={
+          <Text>
+            Do you really want to delete <b>all your instances</b> ?
+          </Text>
+        }
```

Comment:
```suggestion
        alertMessage="Delete ALL instances"
        alertDescription={
          <Text>
            Do you really want to delete <b>all your instances</b> ?
          </Text>
        }
```

Reply:
This modification have been deleted because is part of a @cannarelladev's PR